CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+- Add `kubernetes nodegroup show` for displaying node-group details. This also adds _Nodes_ table and _Anti-affinity_ field that were not available in previous `kubernetes show` output.
```

Comment:
Not a blocker, but just wondering would `node-group` be more aligned with HTTP API resource and SDK's type names :thinking:

Reply:
Yeah, we could go trough the sub-command naming. We have at least `loadbalancer` vs. `load-balancer` as well. Adding aliases would changing the naming while keeping commands backwards compatible.